src/families/crypto_org/js-signOperation.js

```diff
@@ -0,0 +1,165 @@
+// @flow
+import { BigNumber } from "bignumber.js";
+import { Observable } from "rxjs";
+import { FeeNotLoaded } from "@ledgerhq/errors";
+import {
+  CryptoOrgWrongSignatureHeader,
+  CryptoOrgSignatureSize,
+} from "./errors";
+import type { Transaction } from "./types";
+import type { Account, Operation, SignOperationEvent } from "../../types";
+
+import { open, close } from "../../hw";
+import { encodeOperationId } from "../../operation";
+import CryptoOrgApp from "@ledgerhq/hw-app-cosmos";
```

Comment:
Is it ok to use the hw js bindings for the cosmos app ?
Wouldn't be more future-proof to use a cryptoorg hw app, since we don't know if cosmos app would always follow the same lifecycle ?

Reply:
I think for now it is acceptable. CryptoOrg is based on cosmosSDK so it is unlikely that the implementation will diverge, if so we will create a new binding in the future